pyproject.toml

```
@@ -1,3 +1,31 @@
+[project]
+name = "self-signed-certificates-operator"
+version = "0.1.0"
```

What's this version?

---

It is a dummy field, as it is required by the `pyproject.toml` specification. I could also set it to dynamic by doing this instead:

```suggestion
dynamic = ["version"]
```

In any case, it does not impact the resulting artifact.